README.md

```diff
@@ -58,7 +58,78 @@ To build the documentation locally, run `yarn install && yarn build && yarn docs
 in this repository. This will generate a `./docs` directory in each package that you
 can browse locally to see API docs on the various packages.
 
-## Developing under Windows 10
+## Contributing
+
+We are more than happy to accept open source contributions. However, please try
+to work on existing issues or create an issue and get feedback from one of the
+[main contributors](https://github.com/iov-one/iov-core/graphs/contributors)
+before starting on a PR. If you don't know where to start, we try to tag
+["good first issues"](https://github.com/iov-one/iov-core/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+that provide a nice way to get started with the iov-core repo.
+
+### Development Environment
+
+If you go into a subpackage and try to `yarn build` or `yarn test`, chances are it will fail.
+The reason is that we only check in `*.ts` files, while we need the compiled `*.js` files
+to import other packages. We push these to npm but do not check them into git to avoid commit noise.
+
+To get started, please go to the root directory and run:
+
+```
+yarn install
+yarn build
+yarn test
+```
+
+Once that passes, you have the code built and can go into any subdirectory, edit code
+and verify the changes with `yarn test` on that one package.
+
+### Integration Tests
+
+There are a number of integration tests involving communication with tendermint
+(in `iov-tendermint-rpc`) and the bns blockchain (in `iov-bns`) that require
+a test server to run and are skipped by default. If you are working on those
+packages, please run those tests. (They require docker to be installed and
+executable by the current user)
+
+```
+source ./scripts/blockchain_start.sh
+cd packages/iov-tendermint-rpc
+TENDERMINT_ENABLED=1 yarn test
```

Comment:
`TENDERMINT_ENABLED` gets set by `source ./scripts/blockchain_start.sh`

Reply:
ooohhh... so right are you
i should unset that in blockchain_start.sh

Reply:
Yeah, if you want to set TENDERMINT_ENABLED explicitly in the caller, we can run blockchain_start.sh instead of sourcing it.

Reply:
ah, this is what you meant. Alright